src/core/Modal/base.ts

```diff
@@ -0,0 +1,5 @@
+import { ModalProps } from '@material-ui/core/Modal';
+
+type ModalBaseProps = Omit<ModalProps, 'style'>;
```

Comment:
Instead of making a separate base folder, we can do this directly in Modal.tsx because we are dealing with only one component without any type or variant